skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
+        # utility for maintaining previous color image default behavior
+        if ndim == 3:
+            return True
+        else:
+            return False
+
+
 def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
            preserve_range=False):
```

Comment:
Well, both `resize` and `rescale` are exposed to the user. They are used for the same purpose, therefore, I think, they should have the similar API. There is nothing more than that :).

In my opinion, the thinking should be the following: if a function performs any logic on "multichanneling", the argument has to be explicitly provided.